pkgs/development/libraries/spice-gtk/default.nix

```diff
@@ -87,10 +87,13 @@ stdenv.mkDerivation rec {
     vala
   ];
 
+  propagatedBuildInputs = [
+    gst_all_1.gst-plugins-base
```

Comment:
I don't see a reason why this needed to be propagated. Perchance explain?

Reply:
When SpiceGtkClient is loaded via GI it also needs GStreamer and cannot find it because needed paths are not in `GI_TYPELIB_PATH`. I was able to see the cause only after patching `virtManager/details/viewer.py` to show all exceptions.

Reply:
Isn't this only required if spice is enabled? I thin it ahould the same as here, at least applying that change worked for me: https://github.com/NixOS/nixpkgs/blob/7e71cd829290b5a5521f7feef8c0c55d3eb087f0/pkgs/applications/virtualization/virt-viewer/default.nix#L31

Reply:
Ahh nevermind, I can see that you patched spice-gtk directly, so should we remove these additional dependencies also in `virt-viewer`?

Reply:
Yeah, `gst-plugins-base` could be removed, though there's also `gst-plugins-good`.
I'll add a commit after checking if it is needed.